pkg/session_recording/recordings.jsx

```diff
@@ -193,13 +217,77 @@
             });
         }
 
+        sendToTerm(value) {
+            this.refs.terminal.state.terminal.send(value);
+        }
+
+        playPauseToggle() {
+            if (this.state.playback === 'pause') {
+                this.setState({playback: 'play'});
+            } else {
+                this.setState({playback: 'pause'});
+            }
+            this.sendToTerm(' ');
+        }
+
+        speedUp() {
+            if (this.state.speed < 4) {
+                let speed = this.state.speed + 1;
+                this.setState({speed: speed});
+                this.sendToTerm('}');
+            }
```

Comment:
If you're going to assign speed to a variable, might as well save a little more:

    let speed = this.state.speed;
    if (speed < 4) {
        this.setState({speed: speed + 1});
        this.sendToTerm('}');
    }